.gitignore

```diff
@@ -14,6 +14,9 @@ Icon
 # Thumbnails
 ._*
 
+# Code editor files
+.idea
+
```

Comment:
Hi @DiegoGallegos4, please remove this addition to `.gitignore` also. The right way to handle editor-specific files is to add a global `.gitignore` file that applies to your whole computer.

Reply:
See #694 and https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore for more info about this.